bin/commands/down.js

```diff
@@ -42,34 +44,32 @@ exports.builder = (yargs) => {
 
 exports.handler = (argv) => {
   const {
-    spaceId, contentType, dryrun, file
+    spaceId, contentType, dryrun, file, accessToken
```

Comment:
same as above. also I noticed that in a previous commit you changed `dryrun` to `dryRun` should we do the same here?